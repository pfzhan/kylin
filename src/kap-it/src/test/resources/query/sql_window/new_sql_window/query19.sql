select
lag(price,1) OVER ()  lm_active
from
test_kylin_fact
limit 20