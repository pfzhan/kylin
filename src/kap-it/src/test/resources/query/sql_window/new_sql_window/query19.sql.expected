select
lag(price,1) OVER (order by price)  lm_active
from test_kylin_fact
limit 20